labgrid/environment.py

```diff
@@ -23,6 +24,9 @@ def __attrs_post_init__(self):
             raise NoConfigFoundError(
                 "{} is not a valid yaml file".format(self.config_file)
             )
+        for user_file in self.config.get_imports():
+            importlib.import_module(user_file)
+            importlib.invalidate_caches()
```

Comment:
We don't need invalidate_caches, as the file should already exist when starting labgrid.